packages/ember-routing-views/lib/views/link.js

```diff
@@ -421,17 +412,6 @@ var LinkComponent = EmberComponent.extend({
       this.set('disabled', attrs.disabledWhen);
     }
 
-    var currentWhen = attrs['current-when'];
-
-    if (attrs.currentWhen) {
-      Ember.deprecate('Using currentWhen with {{link-to}} is deprecated in favor of `current-when`.', !attrs.currentWhen);
-      currentWhen = attrs.currentWhen;
-    }
-
-    if (currentWhen) {
```

Comment:
Note: I removed this lines, I haven't see the need to duplicate the same attribute under two different keys, both in `this.attrs` and under the object itself (and with different casing).